### PR TITLE
Fix Bug 1357373 - Remove Aurora from Android pre-release channel page

### PR DESCRIPTION
**Pull request by kyoshino**
## Description

* Remove Aurora from the [Android channel page](https://www.mozilla.org/en-US/firefox/channel/android/)
* Redirect the [Android Aurora /all/ download page](https://www.mozilla.org/en-US/firefox/android/aurora/all/) to the [Android Nightly /all/ download page](https://www.mozilla.org/en-US/firefox/android/nightly/all/)

## Bugzilla link

[Bug 1357373](https://bugzilla.mozilla.org/show_bug.cgi?id=1357373)

## Testing

* Open the Android channel page and make sure Aurora is gone.
* Open the Aurora /all/ page and make sure it's redirected to the Nightly /all/ page

## Checklist
- [x] Requires l10n changes.
- [x] Related functional & integration tests passing.

Cc @flodolo for the removal of strings.

### CODE REVIEW

**Comment by alexgibson** [comment]
Flagging this as do-not-merge for now, until we get confirmation that this is in-fact the correct thing to do.

**Comment by kyoshino** [comment]
So the decision is not to remove Developer Edition for desktop. Updated this PR accordingly.

**Comment by pascalchevrel** [comment]
Can we get this merged please? We don't want to expose Aurora content anymore on our web properties now that Aurora no longer exists. Thanks

**Comment by alexgibson** [comment]
> Can we get this merged please? We don't want to expose Aurora content anymore on our web properties now that Aurora no longer exists. Thanks

We're all pretty head down with our main objectives for the quarter. We'll try and get to these PRs soon, sorry for the delay :(

**Comment by alexgibson** [comment]
Please update the functional tests for the `channel/android` page, now that the aurora button is being removed:

https://github.com/mozilla/bedrock/blob/master/tests/functional/firefox/channel/test_android.py
https://github.com/mozilla/bedrock/blob/master/tests/pages/firefox/channel/android.py

File: media/css/firefox/channel.less [hunk 1]
```diff
@@ -507,18 +507,6 @@ html[lang^="en"] {
     }
 }
 
-#aurora header {
-    .at2x('/media/img/firefox/template/aurora-logo.png', 180px, 195px);
```
Comment (alexgibson) [inline media/css/firefox/channel.less]: I think you're safe to remove this image

**Comment by kyoshino** [comment]
Wasn't aware of the functional tests. Will update the PR tonight.

**Comment by kyoshino** [comment]
Got some time up updated the PR, then noticed that the page description contained Aurora. The one string has to be updated (but it may not be a blocker.) Cc @flodolo

**Comment by alexgibson** [comment]
Sorry I missed this earlier, but please remove the retired `android/aurora/all` page from our link checker test file: https://github.com/mozilla/bedrock/blob/master/tests/functional/test_download_l10n.py#L26

File: bedrock/firefox/templates/firefox/channel/android.html [hunk 2]
```diff
@@ -16,7 +16,9 @@
 {% endblock %}
 
 {% block page_desc %}
-{% if l10n_has_tag('firefox_title_meta_bug1345338') %}
+{% if l10n_has_tag('firefox_title_meta_bug1357373') %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/channel/android.html]: I think we should be fine with this string change as we're using an l10n tag 👍

**Comment by alexgibson** [comment]
Looks good @kyoshino just one more small change then I think we should be good to merge 💯

**Comment by kyoshino** [comment]
Removed `/firefox/android/aurora/all/` from `tests/functional/test_download_l10n.py`.

**Pull request merged**